.github/workflows/build_and_test.yml

```diff
@@ -20,35 +18,45 @@ jobs:
         # For every distro we want to test here, add one key 'distro' with a
         # descriptive name, and one key 'containerid' with the name of the
         # container (i.e., what you want to docker-pull)
-        include:
-          - distro: 'Ubuntu 22.04'
-            containerid: 'mormj/newsched-ci-docker:ubuntu-22.04'
+        distro:
+          - name: 'Ubuntu 22.04'
+            containerid: 'ghcr.io/wirew0rm/newsched-ci-docker:ubuntu-22.04'
```

Comment:
@mormj Are you okay with this image being added?  It looks like the original image is in your space.

Reply:
I'd rather just update the image there (see the PR linked above: https://github.com/mormj/newsched-ci-docker/pull/1) to not have the gnuradio ecosystem spread out across multiple github profiles.

Reply:
I haven't tried this yet but we should be able to get the image into ghcr.io/gnuradio - i think it's time to move the dockerfiles from mormj/newsched-ci-docker over to gnuradio-docker

For now i'll update them on dockerhub in mormj but pretty soon should be getting it out of my namespace as well

Reply:
I changed it to use `ghcr.io/mormj/newsched-ci-docker:ubuntu-22.04` which now works after the PR over there was merged, so I guess this can be merged.